tools/image-tag

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
The automation for CD uses `alpine` which doesn't have Bash installed, and it has always used sh. To make sure we continue to support sh I'm explicitly setting it here.